user-interface/src/main/java/life/qbic/datamanager/views/projects/project/measurements/MeasurementDetailsComponent.java

```diff
@@ -186,18 +186,18 @@ private void createNGSMeasurementGrid() {
         .setAutoWidth(true)
         .setFlexGrow(0);
     ngsMeasurementGrid.addComponentColumn(measurement -> {
-          if (measurement.samplePoolGroup().isEmpty()) {
+          if (!measurement.isPooledSampleMeasurement()) {
```

Comment:
Negations are harder to read and understand, especially that "!" is not part our natural language. I suggest to include a method like `isSingleSampleMeasurement()` to reflect the positive case you are looking for.